docs/timeline.rst

```diff
@@ -0,0 +1,61 @@
+.. inclusion-marker-start-do-not-remove
+
+Analyzing Horovod Performance
+=============================
+
+Horovod has the ability to record the timeline of its activity, called Horovod Timeline.
+
+.. image:: https://user-images.githubusercontent.com/16640218/29735271-9e148da0-89ac-11e7-9ae0-11d7a099ac89.png
+   :alt: Horovod Timeline
+
+
+To record a Horovod Timeline, set the `HOROVOD_TIMELINE` environment variable to the location of the timeline
+file to be created.  This file is only recorded on rank 0, but it contains information about activity of all workers.
+
+.. code-block:: bash
+
+    $ HOROVOD_TIMELINE=/path/to/timeline.json horovodrun -np 4 python train.py
+
+
+You can then open the timeline file using the `chrome://tracing` facility of the `Chrome <https://www.google.com/chrome/browser/>`__ browser.
+
+In the example above, you can see few tensors being reduced. There are two major phases for each tensor reduction:
+
+1. **Negotiation** - a phase when all workers send to rank 0 signal that they're ready to reduce the given tensor.
+
+* Each worker reporting readiness is represented by a tick under the *NEGOTIATE_ALLREDUCE* bar, so you can see which workers were early and which were late.
```

Comment:
Indent 3 spaces

Reply:
I changed many things, so I'm not sure if it is fixed. Right now it renders as below. Let me know if I should change anything else, thanks!
Github:
![sample2](https://user-images.githubusercontent.com/42987854/58384449-fd7bc300-7fb7-11e9-8c6b-9d7b2e2afec9.JPG)
Docs:
![sample](https://user-images.githubusercontent.com/42987854/58384454-02407700-7fb8-11e9-857d-fde4fc4af3c2.JPG)